src/components/settings/SettingsLMSTab/LMSConfigs/BlackboardConfig.jsx

```diff
@@ -59,26 +66,31 @@ const BlackboardConfig = ({ id, onClick }) => {
         <Form.Group>
           <Form.Control
             className="my-4"
-            type="text"
+            type="password"
             onChange={(e) => {
               setClientSecret(e.target.value);
             }}
             floatingLabel="API Client Secret/Application Secret"
           />
         </Form.Group>
-        <Form.Group>
+        <Form.Group className="my-4">
           <Form.Control
-            className="my-4"
             type="text"
+            isInvalid={!urlValid}
             onChange={(e) => {
-              setBlackboardBaseUrl(e.target.value);
+              validateUrl(e.target.value);
             }}
             floatingLabel="Blackboard Base URL"
           />
+          {!urlValid && (
+            <Form.Control.Feedback type="invalid">
+              This does not look like a valid url
```

Comment:
suggested: can we have a slightly more informative message (may need to run by UX), but sth like

'URLs needs start with https, have alphanumeric chars etc etc..' whatever makes most sense.

this will also be similar to the other message we have for validation which looks great as in `Display name cannot be over 30 characters`